system_spec/webapi/external_spec.md

```diff
@@ -111,7 +125,7 @@ or
 GET /information?limit=10&tag=api&sort=updated&sortBy=asc
 ```
 
-**PlanB.** "Get old updated 5 information. Tag is 'api' and 'document'."
+**PlanC.** "Get old updated 5 information. Tag is 'api' and 'document'."
```

Comment:
cannot highlight?

"Get old updated 5 information. Tag is 'api' and 'document'."

```"Get old updated 5 information. Tag is 'api' and 'document'."```